packages/client/src/views/Home/RecordAudit.tsx

```diff
@@ -455,84 +402,162 @@ const getApplicationInfo = (
   )
 }
 
-export const ShowRecordAudit = (props: IFullProps) => {
-  const applicationId = props.match.params.applicationId
-  let application: IApplicationData | null
-  application = getSavedApplications(props)
-  const isDownloaded = application ? true : false
-  if (!isDownloaded) {
-    application = getWQApplication(props)
-  }
-
+function RecordAuditBody({
+  clearCorrectionChange,
+  declaration,
+  isDownloaded = false,
+  intl,
+  goToCertificateCorrection
+}: {
+  declaration: IDeclarationData
+  isDownloaded?: boolean
+  intl: IntlShape
+} & IDispatchProps) {
   const actions = []
   if (
     isDownloaded &&
-    application &&
-    application.status === SUBMISSION_STATUS.REGISTERED
+    (declaration.status === SUBMISSION_STATUS.REGISTERED ||
+      declaration.status === SUBMISSION_STATUS.CERTIFIED)
   ) {
     actions.push(
       <TertiaryButton
         id="btn-correct-record"
         align={ICON_ALIGNMENT.LEFT}
         icon={() => <Edit />}
         onClick={() => {
-          props.clearCorrectionChange(applicationId)
-          props.goToCertificateCorrection(
-            applicationId,
-            CorrectionSection.Corrector
-          )
+          clearCorrectionChange(declaration.id)
+          goToCertificateCorrection(declaration.id, CorrectionSection.Corrector)
         }}
       >
-        {props.intl.formatMessage(correctionMessages.title)}
+        {intl.formatMessage(correctionMessages.title)}
       </TertiaryButton>
     )
   }
   return (
-    <div id={'recordAudit'}>
+    <Content
+      title={declaration.name || 'No name provided'}
```

Comment:
Can you please use i18n message here instead of **_No name provided_** that is already defined [here](https://github.com/opencrvs/opencrvs-core/blob/d6325f1a689eb606ac91ef9cd1797959eba912d8/packages/client/src/i18n/messages/constants.ts#L81)